src/msg/async/ProtocolV2.cc

```diff
@@ -585,7 +585,7 @@ bool ProtocolV2::append_frame(F& frame) {
 
   ldout(cct, 25) << __func__ << " assembled frame " << bl.length()
                  << " bytes " << tx_frame_asm << dendl;
-  connection->outgoing_bl.append(bl);
+  connection->outgoing_bl.claim_append(std::move(bl));
```

Comment:
No need to move from `bl` here.

Reply:
i think `std::move()` improves the readability. as it helps us to understand that `bl` does not hold the memory chunk anymore.

Reply:
That is already signaled by "claim" in "claim_append" -- only a couple of places use move with `claim_append()`.